com.stremio.Stremio.metainfo.xml

```diff
@@ -0,0 +1,37 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Ideally you should maintain it in https://github.com/Stremio/stremio-shell but can be handled after the review.

Reply:
I'll move it in future